### PR TITLE
Add `with_bias` parameter to `create_mlp`

### PR DESCRIPTION
**Pull request by qgallouedec**
## Description

It would allow to make `ARSLinearPolicy` a special case of `ARSPolicy`

## Motivation and Context

- [ ] I have raised an issue to propose this change ([required](https://github.com/DLR-RM/stable-baselines3/blob/master/CONTRIBUTING.md) for new features and bug fixes)

(I didn't raise an issue as it seems not to be a big new feature)

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation (update in the documentation)

## Checklist:

- [x] I've read the [CONTRIBUTION](https://github.com/DLR-RM/stable-baselines3/blob/master/CONTRIBUTING.md) guide (**required**)
- [x] I have updated the changelog accordingly (**required**).
- [x] My change requires a change to the documentation.
- [ ] I have updated the tests accordingly (*required for a bug fix or a new feature*).
- [x] I have updated the documentation accordingly.
- [x] I have reformatted the code using `make format` (**required**)
- [x] I have checked the codestyle using `make check-codestyle` and `make lint` (**required**)
- [ ] I have ensured `make pytest` and `make type` both pass. (**required**)
- [ ] I have checked that the documentation builds using `make doc` (**required**)

Note: You can run most of the checks using `make commit-checks`.

Note: we are using a maximum length of 127 characters per line

### CODE REVIEW

**Comment by araffin** [comment]
Do you think it would make sense to include layer norm and dropout in this PR? (see https://github.com/DLR-RM/stable-baselines3/issues/1069, this is to implement REDQ/DroQ as in https://github.com/DLR-RM/stable-baselines3/pull/1036)

or do you prefer if I do that in a separate PR?

**Comment by qgallouedec** [comment]
I would prefer to do it in a separate PR, because you would have to add tests and stuff.

File: stable_baselines3/common/torch_layers.py [hunk 1]
```diff
@@ -98,6 +98,7 @@ def create_mlp(
     output_dim: int,
     net_arch: List[int],
     activation_fn: Type[nn.Module] = nn.ReLU,
+    with_bias: bool = True,
```
Comment (araffin) [inline stable_baselines3/common/torch_layers.py:101]: minor: when adding parameter, I usually prefer to add them at the end (otherwise, it may break some code when not called as keyword argument)

Comment (qgallouedec) [reply stable_baselines3/common/torch_layers.py]: Indeed, it's safer

**Comment by qgallouedec** [comment]
[a60cd1d](https://github.com/DLR-RM/stable-baselines3/pull/1188/commits/a60cd1d75093b899f8d6bf6e0a3948fbb9e7eadf)
Wrong message, read instead "move `with_bias` to the last position"

**Review by araffin — APPROVED** [review]
